main.py

```diff
@@ -1,38 +1,38 @@
 import config
 from utils.experiment import Experiment
 from utils.training_loop import *
-
-
+from training_loop import training_loop_ariel_step
```

Comment:
change name from training_loop_ariel_step, to other name, pay attention that the training loop is per trajectory/epoch and not step.

Reply:
changed to model_training